astropy/config/paths.py

```diff
@@ -159,6 +162,21 @@ def wrapper(*args: P.args, **kwargs: P.kwargs) -> None:
 
         return wrapper
 
+    @staticmethod
+    def _warn_env_var_is_ignored(
+        reason: str, *, env_var_name: str, stacklevel: int = 4
```

Comment:
Given the `*`, `env_var_name` is an optional keyword?
```suggestion
        reason: str, *, env_var_name: str = "", stacklevel: int = 4
```